src/lib/project.js

```diff
@@ -390,36 +382,41 @@ const __dirname = path.dirname(__filename);
 };`
   );
 
-  // This prevents usEffect from running twice on initial mount.
-  newNextConfig = newNextConfig.replace(
-    'reactStrictMode: true',
-    'reactStrictMode: false'
-  );
-
   fs.writeFileSync(path.join('ui', 'next.config.mjs'), newNextConfig);
 
-  const indexFileName = useTypescript ? 'index.tsx' : 'index.js';
+  const pageFileName = 'page.tsx';
+
+  fs.writeFileSync(
+    path.join('ui', 'app', pageFileName),
+    customNextPage,
+    'utf8'
+  );
+
+  const layoutFileName = 'layout.tsx';
 
   fs.writeFileSync(
-    path.join('ui', 'src/pages', indexFileName),
-    customNextIndex,
+    path.join('ui', 'app', layoutFileName),
+    customNextLayout,
     'utf8'
   );
 
   // Adds landing page components directory and files to NextJS project.
   fs.copySync(
     path.join(__dirname, 'ui', 'next', 'components'),
-    path.join('ui', 'src', 'components')
+    path.join('ui', 'components')
   );
 
   // Adds landing page style directory and files to NextJS project.
   fs.copySync(
     path.join(__dirname, 'ui', 'next', 'styles'),
-    path.join('ui', 'src', 'styles')
+    path.join('ui', 'styles')
   );
 
   // Removes create-next-app assets
   fs.emptyDirSync(path.join('ui', 'public'));
+  if (fs.existsSync(path.join('ui', 'app', 'favicon.ico'))) {
```

Comment:
Similar comment to above. 
It looks like there are some legacy `create-next-app` boilerplate styles and fonts leftover in the app folder when a project is generated.  You could empty the entire `app` directory instead of just removing the favicon here with something like `fs.emptyDirSync` or similar before adding the customNextPage and any other files to the directory.